_rules/first-focusable-element-is-link-to-main-8a213c.md

```diff
@@ -1,150 +1,255 @@
+
+## Assumptions
+
+This rule assumes that any global dismissible information that only appears once per site has already been acknowledged and is not displayed any more.
+
+**Note**: The aim of such link is to be able to skip repeated content (headers, navigation bar, ...) when viewing several pages of the same site. Many sites display a cookies policy banner which might be stealing focus until dismissed (usually be viewing and accepting cookies policy). Since that content is _not_ repeated (is it only shown once for the full site), it is not a problem to have it, and it may appear on any page of the site (depending where the user first comes in).
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [G1: Adding a link at the top of each page that goes directly to the main content area](https://www.w3.org/WAI/WCAG21/Techniques/general/G1)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The link to skp repeated content is visible, is included in the accessibility tree, and links to a valid HTML element.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<nav>
+			<a href="#main-content">Skip to main content</a>
+		</nav>
+		<!-- Repeated Content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+The link to skip repeated content can be toggled to visible through keyboard focus.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<style>
+			#skipNav a {
+				height: 100px;
+				position: absolute;
+				top: -100px;
+			}
+			#skipNav a:focus {
+				top: 0px;
+			}
+		</style>
+		<nav id="skipNav">
+			<a href="#main-content">Skip to main content</a>
+		</nav>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Passed Example 3
+
+The link to skip repeated content has an `aria-label` to provide an accessible name.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<nav>
+			<a href="#main-content" aria-label="Skip to main content"></a>
+		</nav>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+There is no link to skip repeated content.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+The link to skip repeated content is not the first focusable element within the `body`.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<a href="/anotherpage">A link to another page</a>
```

Comment:
Putting a dead link in examples is not so good. Can easily be replaced with a link to WCAG, W3C, or whatever other cool resource 😎

Reply:
Sorry! I will add a link to a real external resource 😄